cms/core/models/snippets.py

```diff
@@ -53,3 +58,61 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
     def __str__(self) -> str:
         return str(self.name)
+
+
+class GlossaryTerm(TranslatableMixin, PreviewableMixin, RevisionMixin, index.Indexed, models.Model):
+    """A model for glossary terms."""
+
+    title = models.CharField(max_length=255)
+    definition = RichTextField(features=settings.RICH_TEXT_BASIC)
+    # to allow for a user to be set on creation and seen in the IndexView
+    updated_by = models.ForeignKey(
+        "users.User",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="glossary_terms",
+    )
+    # See https://docs.wagtail.org/en/stable/advanced_topics/reference_index.html
+    updated_by.wagtail_reference_index_ignore = True  # type: ignore[attr-defined]
+
+    revisions = GenericRelation("wagtailcore.Revision", related_query_name="glossary_term")
+
+    panels: ClassVar[list["Panel"]] = [
+        "title",
+        "definition",
+    ]
+
+    search_fields: ClassVar[list[index.BaseField]] = [
+        *index.Indexed.search_fields,
+        index.SearchField("title"),
+        index.AutocompleteField("title"),
+        index.SearchField("definition"),
+        index.AutocompleteField("definition"),
+    ]
+
+    class Meta:
+        verbose_name = _("glossary term")
+        verbose_name_plural = _("glossary terms")
```

Comment:
```suggestion
```

This is one of those cases where you don't need it

https://docs.djangoproject.com/en/5.1/ref/models/options/#verbose-name / https://docs.djangoproject.com/en/5.1/ref/models/options/#verbose-name-plural

Reply:
[Remove verbose name](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/5749ee9196c1d1f8f2b2579d14c916cbe5c8d4a6) - also removed the singular verbose_name for ContactDetails snippet